.travis.yml

```
@@ -31,9 +31,15 @@ jobs:
         - npm run image:build-and-test -- --concurrency 1 --since origin/$TRAVIS_BRANCH || travis_terminate 1
         - npm run image:push -- --since origin/$TRAVIS_BRANCH -- -- --latest=`[ "$TRAVIS_BRANCH" = "$TRUNK_BRANCH" ] && echo true || echo false` || travis_terminate 1
         - git push --follow-tags --no-verify origin $TRAVIS_BRANCH || travis_terminate 1
-    - stage: deploy
-      if: (branch = env(TRUNK_BRANCH) OR branch =~ /^(\d+\.)?\d+-stable$/) AND type = push
+    - stage: deploy to dev
+      if: (branch = env(TRUNK_BRANCH) AND type = push
+      script:
+        - ./ci/prepare-deploy.sh || travis_terminate 1
+        - npm run chart:update -- -- -- --env=dev || travis_terminate 1
+        - npm run chart:deploy -- -- -- --env=dev || travis_terminate 1
```

Isn't that enough for this script?
```suggestion
        - npm run chart:deploy -- --env=dev || travis_terminate 1
```